.github/issue_add_label_config.yml

```diff
@@ -1,27 +1,27 @@
 crash:
-    - "Crash or freeze"
+  - "Crash or freeze"
```

Comment:
These are pure whitespace changes that it would be perfectly reasonable to hide from the blame.